elasticsearch/run.sh

```diff
@@ -24,6 +27,31 @@ retry=$RETRY_COUNT
 max_time=$(( RETRY_COUNT * RETRY_INTERVAL ))	# should be integer
 timeouted=false
 
+# modify configs as needed
+mkdir -p ${HOME}/config
+cp ${ES_CONF}/* ${HOME}/config
+pushd ${HOME}/config
+  if [ -f ${HOME}/config/logging.yml ] && ! grep -q "io\.openshift\.log4jextras" ${HOME}/config/logging.yml ; then
```

Comment:
use `logging.yml` instead of `${HOME}/config/logging.yml` since you already did the pushd, and to be consistent with the python script